### PR TITLE
ISPN-7312 Update KUBE_PING to the latest version

### PR DESCRIPTION
**Pull request by slaskawi**
https://issues.jboss.org/browse/ISPN-7312

### CODE REVIEW

File: parent/pom.xml [hunk 1]
```diff
@@ -173,7 +173,7 @@
       <version.webdav.servlet>2.0.1</version.webdav.servlet>
       <version.weld>2.3.4.Final</version.weld>
       <version.weld-se>2.3.4.Final</version.weld-se>
-      <version.groups.kubernetes>0.9.0</version.groups.kubernetes>
+      <version.groups.kubernetes>0.9.1</version.groups.kubernetes>
```
Comment (gustavonalle) [inline parent/pom.xml]: Was it meant to be  ```version.jgroups.kubernetes``` instead?

Comment (slaskawi) [inline parent/pom.xml]: Yes, let me update it

**Comment by slaskawi** [comment]
@gustavonalle Fixed.

**Comment by gustavonalle** [comment]
merged, thanks @slaskawi

**Pull request merged**